### ISSUE TITLE
SD button not working in protocol 1[BUG]

### ISSUE THREAD

**Issue by hp3ak** [ramdor/Thetis#440]
**Describe the bug**
A clear and concise description of what the bug is. Stereo Diversity (SD button) does not work.  No sync of band and mode and fx, and no separate audio streams to stereo headphones. 
**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Console and activate RX2'
2. Click on 'SD button'
3. Scroll down to 'Notice that Rx2 does NOT change to same band and frequency as Rx1 '
4. See error -- listen with stereo headphones and there are NOT separate audio streams for RX1 & RX2, like in OpenHPSDR mRX PS v3.4.9

**Expected behaviour**
In old software it worked:  Rx2 should change to same band, frequency and mode as Rx1.  Separate audio streams from Rx 1 and Rx 2 should appear in stereo headphones.
**Screenshots**
SD button lights, but no one is home.
**Desktop (please complete the following information):**
 - OS: Windows 7, 24 GB ram.  Anan 100D with firmware version 5.9 (protocol 1)

**Additional context**
Works fine in mRX PS v3.4.9.  I don't believe that it has ever worked properly in Thetis and not at all now.

**Comment by RdWing** [comment]
Diversity in general is a bit buggy, even with Protocol 2.

**Comment by ramdor** [comment]
there is no code behind the button in Thetis 2.10.3.6. I will hide the button for now until a decision is made to re-implement it. Will leave this as a feature, not a bug.

**Comment by hp3ak** [comment]
Thanks for noticing.  I do not consider the inability to use Rx2 for diversity receive a "feature enhancement". However, I guess that getting the real functionality of having two, independent phase locked receivers has never been a priority in the Thetis branch of the software. I have tried almost every new release of Thetis and no joy.

**Comment by ramdor** [comment]
I looked at OpenHPSDR-PowerSDR v3.4.9 from the 20th May 2018. Specifically I looked at console.cs., and this was found :

SD does the following...

rx2 DSP mode is copied from rx1
rx2 filter is copied from rx1
puresignal rx2 freq is set to the dds frequency of rx1 (this is now different)
when an oscillator shift is made on rx1, the same is made to rx2
vfoB is copied to vfoA

In short, It is not exactly difficult to do without the SD button in Thetis. Just swich on VFOSync, set your filters accordingly.

**Comment by ramdor** [comment]
two different antennas, panned left for rx1, right for rx2. Working perfectly.

![image](https://github.com/ramdor/Thetis/assets/3124967/ae0eef58-c1cc-470f-98d5-c821140f1a58)

**Comment by hp3ak** [comment]
Yes, it can be done with VFO Sync, but with different functionality..    

*** With VFO Sync ON, clicking VFO B [TX]  & changing the tx fx for split operation now changes fx of Rx2, losing the diversity reception.  **Using SD with PowerSDR 3.4.9, the two receivers stay locked even when VFO B is chosen for TX and shifted for split.**  So, unfortunately not the same thing.

Note:  I spent 10 years testing and managing testing of consumer software in Silicon Valley.  I am not a programmer.  If so I would try to fix the long list of frustrating operational bugs that remain in Thetis.  Like losing the NR2 selection when shifting from DIGU to LSB/USB on the same band.  Or losing all of the frequencies in a band register when Importing a saved database.  Or, mysteriously losing all custom Equalizer settings for Rx and Tx.  Or changing the mode in one of the 5 band registers.  Or overwriting fx 4 register with the contents of register 5 at times.   I could go on, but there is no point.  I will keep trying new releases of Thetis as they come out, hoping that someday it will be ready for Prime Time.  Sorry, that is not the case presently.
